README.md

```diff
@@ -54,6 +54,9 @@ import { Picker } from 'emoji-mart'
 | **notFoundEmoji** | | `sleuth_or_spy` | The emoji shown when there are no search results |
 | **notFound** | | | [Not Found](#not-found) |
 | **icons** | | `{}` | [Custom icons](#custom-icons) |
+| **alwaysShowSkinTones** | | `false` | Always display skin tones. `showPreview` should be set to `false` when enabled.
+| **showAddCustomEmojiButton** | | `false` | Display `Add Emoji` button. `alwaysShowSkinTones` should be set to `true` when enabled.
+| **onAddCustomEmoji** | | | `Add Emoji` button's `onClick` fallback.
```

Comment:
I was unsure about adding these to the `README` since for some reason, we didn't have the `alwaysShowSkinTones` prop there in the first place. Please let me know if these shouldn't be out, and I'll remove them.